packages/vc-verification/src/verifier/revocation-verification.ts

```diff
@@ -238,14 +248,9 @@ export class RevocationVerification {
             await this._statusListEntryVerificaiton.verifyCredentialStatus(
               issuerCredential.credentialStatus
             );
-          } else {
-            const rolePayload =
-              await this.claimIssuerVerification.verifyIssuance(
-                issuer,
-                issuers?.roleName,
-                roleCredentialCache
-              );
```

Comment:
I think this is the implementation of point "1. Remove issuance verification check from revocation status check", right?
Could you explain how this affects the way in which the client trusts `StatusListCredential` @nichonien ? It seems to me this would open the possibility of the status list being inauthentic. Is it that the client is trusting that the server has done this issuance check and is serving a valid `StatusListCredential`?

Reply:
@jrhender We perform this validation (issuance) in the verifyIssuer call also (which is just followed by this revocation check - https://github.com/energywebfoundation/ew-credentials/blob/08461849f69eb288e276c45f874e052eff386047/packages/vc-verification/src/verifier/issuer-verification.ts#L104). So, having the same here in revocation is just redundant check and it is best to only check the status (expiration and revocation) in this part of the verification.

Reply:
Removed this change from this PR